var/spack/repos/builtin/packages/z-checker/package.py

```diff
@@ -12,10 +12,18 @@ class ZChecker(AutotoolsPackage):
     """a library to perform the compression assessment for lossy compressors"""
 
     homepage = "https://github.com/CODARcode/Z-checker"
-    url      = "https://github.com/CODARcode/Z-checker/releases/download/0.5.0/Z-checker-0.5.0.tar.gz"
-
+    url      = "https://github.com/CODARcode/Z-checker/releases/download/0.6.0/Z-checker-0.6.0.tar.gz"
+    git      = "https://github.com/CODARcode/Z-checker"
     maintainers = ['disheng222']
 
+
+    def url_for_version(self, version):
+        """provide url to ensure that download counting via github releases
+        works accurately"""
+        url = "https://github.com/CODARcode/Z-checker/releases/download/{0}/Z-checker-{0}.tar.gz"
+        return url.format(version)
```

Comment:
Can you move the function below, under all the directives?

Reply:
Hi alalazo, 
I just committed a new version based on your suggestion. I guess I don't need to submit a new pull request, do I?
Thanks.

Reply:
No, that's absolutely fine to continue in this PR.